bitnami/kube-prometheus/values.yaml

```diff
@@ -796,10 +796,10 @@ prometheus:
   tolerations: []
   ## @param prometheus.scrapeInterval Interval between consecutive scrapes
   ##
-  scrapeInterval: ""
+  scrapeInterval: "1m"
   ## @param prometheus.evaluationInterval Interval between consecutive evaluations
   ##
-  evaluationInterval: ""
+  evaluationInterval: "1m"
```

Comment:
How are the changes to those default values related with the issue? I also see that in both cases, we are only setting the variables if they are not empty.